context.go

```diff
@@ -0,0 +1,10 @@
+package telebot
+
+const Default string = ""
```

Comment:
I guess dont know whether the concept of a default handler itself is beneficial or is just overcomplicating things with no real value for the user. I just imported the idea form your PR without thinking too much about it and now it does not feel right

I think for now I will stick to an MVP approach. If route() can not find any valid handler, no handler is executed